Makefile

```diff
@@ -453,19 +454,19 @@ ifneq ($(HIP_LIB_DIR),)
 endif
 
 # SYCL Backends
-SYCL_BACKENDS = /gpu/sycl/ref /gpu/sycl/shared
+SYCL_BACKENDS = /gpu/sycl/ref /gpu/sycl/shared /gpu/sycl/gen
 ifneq ($(SYCL_DIR),)
 	SYCL_LIB_DIR := $(wildcard $(foreach d,lib lib64,$(SYCL_DIR)/$d/libsycl.${SO_EXT}))
 	SYCL_LIB_DIR := $(patsubst %/,%,$(dir $(firstword $(SYCL_LIB_DIR))))
 endif
 ifneq ($(SYCL_LIB_DIR),)
 	PKG_LIBS += $(SYCL_FLAG) -lze_loader
 	LIBCEED_CONTAINS_CXX = 1
-	libceed.sycl += $(sycl-core.cpp) $(sycl-ref.cpp) $(sycl-shared.cpp)
+	libceed.sycl += $(sycl-core.cpp) $(sycl-ref.cpp) $(sycl-shared.cpp) $(sycl-gen.cpp)
 	BACKENDS_MAKE += $(SYCL_BACKENDS)
 endif
 
-# MAGMA Backends
+# MAGMA Backend
```

Comment:
```suggestion
# MAGMA Backends
```

ie. keep the original code.

Reply:
Yeah, I agree with James. For anyone wondering, we actually do have multiple MAGMA backends even though we only build for one hardware type at a time -- `/gpu/*/magma` and `/gpu/*/magma/det` (deterministic version, uses ref restrictions).